Tweaks to command-line help, readme and contribution guidelines (#9)
IBM/zowe-cli-cics-deploy-plugin: README.md

# zowe-cli-cics-deploy-plugin
This project aims to create a plugin for Zowe CLI https://github.com/zowe/zowe-cli.  The plugin will provide the capability to deploy applications developed on a workstation to IBM CICS, as CICS bundles. Initially it will support deploying Node.js applications.  It aims to provide an experience similar to deploying to a cloud platform when deploying to CICS.  It will also provide low-level commands for performing individual pieces of the deployment process that could be used as part of a CI/CD pipeline.

## Status

> It's currently a work in progress at a fairly early stage. If you'd like to try it out, at the moment you'll need to build the plugin from source as described [here](docs/tutorials/Setup.md). 

Reviewer: ```suggestion
It is currently a work in progress at a fairly early stage. If you would like to try it out, you need to build the plugin from source as described [here](docs/tutorials/Setup.md). 
```

## Contributing

> Contributions are welcom, see our [contribution guidelines](CONTRIBUTING.md).  

Reviewer: ```suggestion
Contributions are welcome, see our [contribution guidelines](CONTRIBUTING.md).  
```
